src/Jackett.Common/Indexers/Wolfmax4K.cs

```diff
@@ -0,0 +1,498 @@
+                { "q", query.SearchTerm },
+                { "pgb", page.ToString() }
+            };
+
+            var result = await RequestWithCookiesAndRetryAsync(SiteLink + "buscar", data: body, method: RequestType.POST, emulateBrowser: false);
+            if (result.Status != HttpStatusCode.OK)
+                throw new ExceptionWithConfigData(result.ContentString, configData);
+            return result;
+        }
+
+        private static TorznabQuery SanitizeTorznabQuery(TorznabQuery query)
+        {
+            // Taken from Dontorrent.cs
+            // Eg. Marco.Polo.2014.S02E08
+
+            // the season/episode part is already parsed by Jackett
+            // query.SanitizedSearchTerm = Marco.Polo.2014.
+            // query.Season = 2
+            // query.Episode = 8
+            var searchTerm = query.SanitizedSearchTerm;
+
+            // replace punctuation symbols with spaces
+            // searchTerm = Marco Polo 2014
+            searchTerm = Regex.Replace(searchTerm, @"[-._\(\)@/\\\[\]\+\%]", " ");
+            searchTerm = Regex.Replace(searchTerm, @"\s+", " ");
+            searchTerm = searchTerm.Trim();
+
+            // we parse the year and remove it from search
+            // searchTerm = Marco Polo
+            // query.Year = 2014
+            var r = new Regex("([ ]+([0-9]{4}))$", RegexOptions.IgnoreCase);
+            var m = r.Match(searchTerm);
+            if (m.Success)
+            {
+                query.Year = int.Parse(m.Groups[2].Value);
+                searchTerm = searchTerm.Replace(m.Groups[1].Value, "");
+            }
+
+            // remove some words
+            searchTerm = Regex.Replace(searchTerm, @"\b(espa[ñn]ol|spanish|castellano|spa)\b", "", RegexOptions.IgnoreCase);
+
+            query.SearchTerm = searchTerm;
+            return query;
+        }
+
+        private ReleaseInfo ExtractReleaseInfo(IElement cardElement, TorznabQuery query)
+        {
+            // https://wolfmax4k.com/descargar/peliculas-castellano/bebe-made-in-china-2020-/blurayrip-ac3-5-1/
+            // https://wolfmax4k.com/descargar/la-sala-de-torturas-chinas/
+            // https://wolfmax4k.com/pelicula/el-hombre-de-chinatown/
+            // https://wolfmax4k.com/descargar/documentales/misterios-de-china/temporada-1/capitulo-03/
+            // https://wolfmax4k.com/descargar/serie/the-legend-of-vox-machina/temporada-2/capitulo-11/
+            // https://wolfmax4k.com/descargar/peliculas-x264-mkv/los-tesoros-del-mar-de-china-1987-/bluray-microhd/
+            // https://wolfmax4k.com/descargar/serie/the-legend-of-vox-machina/temporada-1/capitulo-04-al-06/
+            // https://wolfmax4k.com/descargar/serie-en-hd/top-boy/temporada-3/capitulo-02/
+            // https://wolfmax4k.com/descargar/programas-tv/la-isla-de-las-tentaciones/temporada-7/capitulo-10/
+            // https://wolfmax4k.com/descargar/serie-1080p/historial-delictivo/temporada-1/capitulo-02/
+
```

Comment:
some torznab apps do not like getting results with 0 byte sizes. If the site does not provide a size then we usually guess some default size values 
https://github.com/Jackett/Jackett/blob/7c49cea4b64cc95166250d441d94f1692eb00dcf/src/Jackett.Common/Indexers/DonTorrent.cs#L505-L516
https://github.com/Jackett/Jackett/blob/7c49cea4b64cc95166250d441d94f1692eb00dcf/src/Jackett.Common/Indexers/DonTorrent.cs#L527-L541